### PR TITLE
Extract system sound and vibration functionality out of activity.mojom

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

File: sky/services/platform/vibrator.mojom [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module flutter.platform;
+
+/// Allows access to a vibrator on the device. This API is intentionally terse
+/// since it invokes default platform behavior. This interface is not suitable
+/// for use if you require more flexible access to device sensors and
+/// peripherals.
+interface Vibrator {
```
Comment (krisgiesing) [inline sky/services/platform/vibrator.mojom]: Can we call the interface Haptic instead?

File: sky/services/platform/system_sound.mojom [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+module flutter.platform;
+
+/// A sound provided by the system
+enum SystemSoundType {
+  Click,
+};
+
+/// Allows easy access to the library of short system specific sounds for
+/// common tasks.
+interface SytemSound {
```
Comment (krisgiesing) [inline sky/services/platform/system_sound.mojom]: Typo: Sy_s_temSound

File: sky/services/platform/vibrator.mojom (hunk 1 above)
Comment (chinmaygarde) [inline sky/services/platform/vibrator.mojom]: HapticFeedback? Or just Haptic?

Comment (krisgiesing) [inline sky/services/platform/vibrator.mojom]: Yeah I think I like HapticFeedback better.

**Comment by chinmaygarde** [comment]
Another look?

**Comment by krisgiesing** [comment]
LGTM

**Comment by chinmaygarde** [comment]
Rebased

**Pull request merged**